spec/end_of_life/repository_spec.rb

```diff
@@ -108,6 +107,41 @@
       end
     end
 
+    describe "#fetch" do
+      let(:client) { instance_double(Octokit::Client, :user => user) }
```

Comment:
Do you mind updating the tests to not use `let`s and possibly `before`s? I'm [not a big fan](https://thoughtbot.com/blog/lets-not) of them 😅

Reply:
Rewritten, & broken out into two `it`s.